README.md

```diff
@@ -123,15 +129,16 @@ If a Lord drops their hammer during the Flip phase they are instantly demoted to
+If there is at least one Lord and no active Uprising nail currently exists, Lads may call "Uprising" at the start of their turn before flipping the hammer. If they do, they place a single Uprising nail somewhere in the stump and hammer it to the same starting depth as all other nails. Uprising rules now apply to the duration of this turn.
 
-The active player forfeits the ability to give out drinks during an Uprising. However, they still must drink if they miss, regrip, drop the hammer, or are penalized in some other way.
+As long as there is an active Uprising nail and the current player is a Lad, they may call "Uprising" before flipping their hammer. If they do, then the turn officially counts as an Uprising and Uprising rules apply for the rest of the turn. 
 
-If the Uprising counter reaches 3 all Lords must immediately drink. If the Uprising counter reaches 7 a Coup occurs all Lords are immediately demoted to Lads and must drive a new starting nail into the outer ring of the stump. When a Coup occurs the Uprising counter is reset to 0.
+During an Uprising, the active player must perform a double hammer flip followed by one strike on the Uprising nail without hesitation (note that they do not get two strikes like they normally would). If this strikes completely hammers the Uprising nail into the stump, all Lords are immediately demoted to Lad and must begin a new starting nail. If the active player already has their starting nail hammered in (if the amount of Lords was previously the maximum) they then instantly ascend to Lord. The Uprising nail is no now longer active and a new one may be started by future Lads.
 
-If a player would normally become a Lord upon performing their Uprising strike, they do not become a Lord until any Uprising effects for this turn resolve. I.e., If the Uprising counter is at 6 and a player performs a successful Uprising but in the process also completely hammers their starting nail into the stump, all existing Lords would be demoted and then the active player would be promoted to Lord. A player may not give out drinks during an Uprising even if they become Lord.
+A Lad may call uprising and then perform a triple or even higher order flip in which case they may take multiple strikes to the Uprising nail or any other valid targets. However, the first strike must hit the center nail. The amount of strikes a player gets in this case is n - 1 (including the initial Uprising strike) where n is the number of flips.
```

Comment:
"A lad may call `"Uprising"`" for consistency.

I think this could benefit from some rewording as it's a little unclear. What are "any other valid targets"? Why are they valid? What is the "center nail"?

Reply:
I'll reword it a bit. Center nail is a typo too -- I meant Uprising nail